authors/apps/authentication/views.py

```diff
@@ -75,3 +79,60 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ResetPasswordEmail(CreateAPIView):
+    """Password reset emailing with uid and reset token"""
+    permission_classes = (AllowAny,)
+    serializer_class = ResetEmailSerializer
+
+    @classmethod
+    def post(cls, request, *args, **kwargs):
+        """Returns message for reset password email"""
+        data = request.data
+        serializer = cls.serializer_class(data=data)
+        serializer.is_valid(raise_exception=True)
+        user = User.objects.get(email=data['email'])
+        body = {
+            'user': data['email'],
+            'message': 'Please use the uid and token to rest your password',
+            'uid': urlsafe_base64_encode(force_bytes(data['email'])).decode(),
+            # 'token': default_token_generator.make_token(user),
+            'token': password_rest_token.make_token(user),
+        }
+        from_email = settings.DEFAULT_FROM_EMAIL
+        to_email = data['email']
+        subject = 'Confirm Your Article Account Password Reset'
+        try:
+            send_mail(subject, body, from_email, [to_email], fail_silently=False)
+            response = {'message': 'check your email for the rest password token',
+                        'status_code': status.HTTP_200_OK}
+        except Exception:
+            response = {'Connection error': 'Check your internet connection',
+                        'status_code': status.HTTP_503_SERVICE_UNAVAILABLE}
+        return Response(response, content_type='text/json')
+
+
+class ResetPasswordConfirm(RetrieveUpdateAPIView):
```

Comment:
@ManuelDominic, it's better  practice to use a generic view which only performs what you require. For example here, the best would be `UpdateAPIView`